packages/legend-manual-tests/src/__tests__/roundtrip-grammar-mismatch/cases/DSL_Mapping-basic-class-mapping-extends-implicit-to-explicit.pure

```diff
@@ -0,0 +1,558 @@
+--- BEFORE ---
```

Comment:
Currently the protocol is not identical on the way in and out as we move the legacy style includes to the new includes. We are making this change in engine also so that it upconverts this...then we can move them back. 
The exact match test cases check the protocol round trips...so instead I moved them here.

Reply:
See the block of code in grammar round trip test, theres a way to leave the test there but indicate which phase to skip, in this case, its protocol check

Reply:
Sweet thanks. Changed.